qiskit_ibm_runtime/options/resilience_options.py

```diff
@@ -100,8 +100,15 @@ def _validate_options(self) -> "ResilienceOptionsV2":
         # Validate not ZNE+PEC
         if self.pec_mitigation is True and self.zne_mitigation is True:
             raise ValueError(
-                "pec_mitigation and zne_mitigation`options cannot be "
+                "'pec_mitigation' and 'zne_mitigation' options cannot be "
                 "simultaneously enabled. Set one of them to False."
             )
 
+        if not isinstance(self.layer_noise_model, UnsetType) and None in [
```

Comment:
I had that, but I get a mypy error here because "self.layer_noise_model is not iterable." What you suggest here works in practice, but I couldn't find a way of disabling the mypy complaint